app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,13 @@
           window.location = searchUrl;
       }
   </script>
```

Comment:
I'd pull this into a separate erb file (not because it will get referenced anywhere else, just because it will declutter this file)

Reply:
Good point, done in beb0ff9.

Reply:
oh nice!  I was thinking of pulling out the whole script tag, but I actually like what you did better, since the separate file is now pure js.